src/effects/effectslot.cpp

```diff
@@ -313,13 +293,11 @@ void EffectSlot::loadEffect(const EffectManifestPointer pManifest,
     unsigned int iParameterSlot = 0, iButtonParameterSlot = 0, iNumParameter = 0;
     for (const auto& pManifestParameter: m_pManifest->parameters()) {
         if (isKnobParameter(pManifestParameter)) {
-            m_knobParameterSlots[iParameterSlot]->loadManifestParameter(
+            m_knobParameterSlots[iParameterSlot++]->loadManifestParameter(
```

Comment:
Please do not use this antifeature of C++ that lets you assign to a variable and reference it on the same line. It makes the code unnecesarily difficult to think about. Put `++iParameterSlot;` back on its own line.

Reply:
Updated.